src/MiniExcel/MiniExcelLibs.csproj

```diff
@@ -2,6 +2,7 @@
 	<PropertyGroup>
 		<TargetFrameworks>net45;netstandard2.0;net5.0</TargetFrameworks>
 		<Version>1.23.2</Version>
+		<PackageVersion>1.23.2.1</PackageVersion>

```

Comment:
Probably none of `PackageVersion`, `PackageId`, `PackageProjectUrl`, `RepositoryUrl` and `PackageReleaseNotes` should be changed as part of this pull request.

Reply:
sorry. I removed.